contracts/src/CrossChainApplications/NativeTokenBridge/tests/ERC20TokenSourceTests.t.sol

```diff
@@ -17,28 +17,14 @@ import {
 } from "../ERC20TokenSource.sol";
 import {UnitTestMockERC20} from "../../../Mocks/UnitTestMockERC20.sol";
 
-contract ERC20TokenSourceTest is Test {
-    address public constant MOCK_TELEPORTER_MESSENGER_ADDRESS =
-        0x644E5b7c5D4Bc8073732CEa72c66e0BB90dFC00f;
-    address public constant WARP_PRECOMPILE_ADDRESS =
-        address(0x0200000000000000000000000000000000000005);
-    bytes32 private constant _MOCK_BLOCKCHAIN_ID = bytes32(uint256(123456));
-    bytes32 private constant _DEFAULT_OTHER_CHAIN_ID =
-        bytes32(hex"abcdefabcdefabcdefabcdefabcdefabcdefabcdefabcdefabcdefabcdefabcd");
-    address private constant _DEFAULT_OTHER_BRIDGE_ADDRESS =
-        0xd54e3E251b9b0EEd3ed70A858e927bbC2659587d;
-    uint256 private constant _DEFAULT_INITIAL_RESERVE_IMBALANCE = 1000000000;
-    address private constant _DEFAULT_RECIPIENT = 0xa4CEE7d1aF6aDdDD33E3b1cC680AB84fdf1b6d1d;
-    uint256 private constant _DEFAULT_TRANSFER_AMOUNT = 1e18;
-    uint256 private constant _DEFAULT_FEE_AMOUNT = 123456;
-
+contract ERC20TokenSourceTest is NativeTokenBridgeTest {
```

Comment:
Ohhh I was wondering if there was a way to share code between these tests. Thank you 🙏 . I think there's some setup/mocking code they share as well, but I can clean that up after.